packages/ui-styles/scss/components/dropdown/_dropdown.scss

```
@@ -0,0 +1,58 @@
+@import "../../lib/colors";
+@import "../../lib/fonts";
+@import "../../lib/mixins";
+@import "../../lib/variables";
+
+@mixin dropdown {
+  &.dropdown {
+    color: $color-asu-maroon;
+    background: $color-white;
+    width: 10rem;
+    border-radius: $dropdown-border-radius;
+    border: 0.3rem solid transparent;
+
+    &:hover {
+      border-color: $color-light-grey;
```

I've added colors from our ASU color palette on Figma to `packages\ui-styles\scss\lib\_colors.scss`. Use the newly added colors. Reference Figma for which colors to use.
```suggestion
      border-color: $color-gray-5;
```

---

I did the changes and also I've updated some colors in _button.scss as there were some properties which were using old color variables.